packages/docs/src/data/showcase-videos.tsx

```diff
@@ -1245,6 +1245,26 @@ export const showcaseVideos: ShowcaseVideo[] = [
 		"name": "Yuwan Kumar"
 	}
 },
+	
+{
+	title: "Blast Workout Rotato style 3D",
+	type: "mux_video",
+	muxId: "pw7GlBHgVlhwE9JMWMXk5ta01n2pPBEIgu8akS025zYB8",
+	description: "Blast Workout Rotato style 3D showcasing app features",
+	height: 1280,
+	width: 720,
+	submittedOn: new Date("09-30-2022"),
+	links: [
+		{
+			type: "website",
+			url: "https://blastworkout.app",
+		},
+	],
+	author: {
+		"url": "https://blastworkout.app",
+		"name": "Mad Mustache Company>"
```

Comment:
Is this deliberate that there is a `>` at the end?
I also saw it in the other PR and removed it there 🤔

Reply:
Oups definitely not deliberate, thanks a lot for removing it in the previous PR, I'll pay more attention next time.